bridge/whatsapp/handlers.go

```diff
@@ -88,11 +91,75 @@ func (b *Bwhatsapp) HandleTextMessage(message whatsapp.TextMessage) {
 	b.Remote <- rmsg
 }
 
-//
-//func (b *Bwhatsapp) HandleImageMessage(message whatsapp.ImageMessage) {
-//	fmt.Println(message) // TODO implement
-//}
-//
+// HandleImageMessage sent from WhatsApp, relay it to the brige
+func (b *Bwhatsapp) HandleImageMessage(message whatsapp.ImageMessage) {
+	if message.Info.FromMe { // || !strings.Contains(strings.ToLower(message.Text), "@echo") {
+		return
+	}
+
+	// whatsapp sends last messages to show context , cut them
+	if message.Info.Timestamp < b.startedAt {
+		return
+	}
+
+	messageTime := time.Unix(int64(message.Info.Timestamp), 0) // TODO check how behaves between timezones
+	groupJid := message.Info.RemoteJid
```

Comment:
Should be `groupJID`. I know we can't influence the struct field's name as it's third-party but we should aim to keep to idiomatic Go in our own codebase.